pkg/ops/iso.go

```diff
@@ -199,7 +202,13 @@ func InjectISO(dst, isoFile string, i schema.ISO) func(ctx context.Context) erro
 		if err != nil {
 			return err
 		}
-		internal.Log.Logger.Info().Msgf("Wrote '%s'", injectedIso)
+		os.Remove(isoFile)                    // remove the original iso file
```

Comment:
instead of keeping 2 isos, we keep only one the injected one, if you want a clean iso, dont pass a cloud config, solved, wtf are you even doing bro?